CHANGELOG.md

```diff
@@ -1,5 +1,51 @@
 # PowerShell Extension Release History
 
+## v2020.9.0-preview
+### Wednesday, September 16, 2020
+#### [vscode-PowerShell](https://github.com/PowerShell/vscode-PowerShell)
+
+- 📺✨ [vscode-powershell #2919](https://github.com/PowerShell/vscode-powershell/pull/2919) -
+  Untitled Notebooks and Automatic Notebooks.
+- 📺✨ [vscode-powershell #2935](https://github.com/PowerShell/vscode-powershell/pull/2935) -
+  Add ISE theme settings borders. (Thanks @CalistusK!)
+- 👷✨ [vscode-powershell #2922](https://github.com/PowerShell/vscode-powershell/pull/2922) -
+  Add editorconfig to recommended extensions for the local build. (Thanks @xtqqczze!)
+- 👷✨ [vscode-powershell #2889](https://github.com/PowerShell/vscode-powershell/pull/2889) -
+  Use spaces instead of tabs for consistency. (Thanks @xtqqczze!)
+- 👷✨ [vscode-powershell #2884](https://github.com/PowerShell/vscode-powershell/pull/2884) -
+  Update copyright header in *.ps1, *.psm1 files. (Thanks @xtqqczze!)
+- 🐢🐛 [vscode-powershell #2888](https://github.com/PowerShell/vscode-powershell/pull/2888) -
+  Fixed crash when running pester older than 3.4.5. (Thanks @EmmanuelPineiro!)
+- 👷✨ [vscode-powershell #2881](https://github.com/PowerShell/vscode-powershell/pull/2881) -
+  Remove UTF-8 BOM from misc text files. (Thanks @xtqqczze!)
+- 👷✨ [vscode-powershell #2876](https://github.com/PowerShell/vscode-powershell/pull/2876) -
+  Update text in LICENSE.txt. (Thanks @xtqqczze!)
+- 📺✨ [vscode-powershell #2861](https://github.com/PowerShell/vscode-powershell/pull/2861) -
+  Update LSP Client to use new Semantic Highlighting APIs. (Thanks @justinytchen!)
+- 👷✨ [vscode-powershell #2871](https://github.com/PowerShell/vscode-powershell/pull/2871) -
+  Prevent `Update Notebook dts` action from triggering on forked repositories. (Thanks @corbob!)
+- 🔧🐛 [vscode-powershell #2863](https://github.com/PowerShell/vscode-powershell/pull/2863) -
+  Fix migration of `codeFormatting.addWhitespaceAroundPipe` setting.
```

Comment:
Maybe worth mentioning that this won't add this setting when it doesn't already exist now

Reply:
```suggestion
  Fix migration of `codeFormatting.addWhitespaceAroundPipe` setting when it doesn't already exist.
```